executor/common_linux.h

```diff
@@ -4534,10 +4534,11 @@ static void remove_dir(const char* dir)
 	DIR* dp = 0;
 retry:
 #if SYZ_EXECUTOR || !SYZ_SANDBOX_ANDROID
+	const int umount_flags = MNT_FORCE | UMOUNT_NOFOLLOW;
```

Comment:
Please add a comment here why we use options and what options we don't want to use and why. In several years somebody will restore MNT_DETACH because the mount is hanging.

Btw, I suspect it will hang because something keeps references to something.

Reply:
> Btw, I suspect it will hang because something keeps references to something.

We could call remove_dir() before reply_execute(), then maybe hanging remove_dir() will result in some error that will be noticed by the fuzzer. However it will increase test latency, so not sure if it's a good idea.

Reply:
I've added a comment.

We can try to deploy it on syzbot and see it has affected the speed of fuzzing in any way. If there's any dramatic effect, we'd surely notice.